validator/accounts/account.go

```diff
@@ -223,7 +223,15 @@ func HandleEmptyKeystoreFlags(cliCtx *cli.Context, confirmPassword bool) (string
 
 // Merge merges data from validator databases in sourceDirectories into a new store, which is created in targetDirectory.
 func Merge(ctx context.Context, sourceDirectories []string, targetDirectory string) error {
-	var sourceStores []*db.Store
+	var sourceStoreSlice = make([]*db.Store, 0)
+	var sourceStores = &sourceStoreSlice
```

Comment:
why do we need to deference `sourceStoreSlice`?

Reply:
My bad. I misunderstood how `defer` works. I'll need to do some more reading about this. It should be fixed now.